central/reports/scheduler/v2/reportgenerator/report_gen_impl_test.go

```diff
@@ -273,6 +274,7 @@ func (s *EnhancedReportingTestSuite) TestGetReportData() {
 			s.ElementsMatch(tc.expected.imageNames, reportData.imageNames)
 			s.ElementsMatch(tc.expected.componentNames, reportData.componentNames)
 			s.ElementsMatch(tc.expected.cveNames, reportData.cveNames)
+			s.Equal(len(tc.expected.cveNames), len(reportData.cvss))
```

Comment:
I just wanted to add assertion that reportData has csvv score for every cve. I am not sure about expected csvv value

Reply:
Oh I meant compare the length of `tc.expected.cvss` in the expected value instead of `tc.expected.cveNames`. But what you have makes sense too. So you can keep it the same.